.github/actions/install_dbt_requirements/action.yaml

```diff
@@ -5,10 +5,6 @@ inputs:
     description: Path to the directory containing the dbt project.
     required: false
     default: ./dbt
-  requirements_file_path:
-    description: Path to Python requirements file.
-    required: false
-    default: ./dbt/requirements.txt
```

Comment:
Is there a context in which this is ever called? I couldn't find one so I removed it in favor of just setting a working directory.

Reply:
**[Praise]** I think it's fine to consolidate this with the working directory. I was just being a bit overly defensive following [GitHub's advice to define all filepaths as environment variables](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables).